docker/tcpv4tracer.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python
+#
+# tcpv4tracer   Trace TCP IPv4 connections.
+#               For Linux, uses BCC, eBPF. Embedded C.
+#
```

Comment:
What kind of reference do you mean exactly? BTW this whole file, including this section, is going to change with @iaguis's new tracer.